### PR TITLE
kubevirt: bump kubevirt-web-ui-components to v0.1.45

### PR DESCRIPTION
**Pull request by mareklibra**
Change log: https://github.com/kubevirt/web-ui-components/releases/tag/v0.1.45

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "@console/internal": "0.0.0-fixed",
     "@console/plugin-sdk": "0.0.0-fixed",
     "@console/shared": "0.0.0-fixed",
-    "kubevirt-web-ui-components": "~0.1.44"
+    "kubevirt-web-ui-components": "~0.1.45"
```
Comment (vojtechszocs) [inline frontend/packages/kubevirt-plugin/package.json]: FYI @mareklibra we don't have to modify the semver range if the new version matches the existing range.

`~0.1.44` matches `0.1.44`, `0.1.45`, `0.1.46` etc.

I'd suggest doing version bumps via `yarn upgrade-interactive` if possible, this should only bump the resolution entry in `yarn.lock` file without touching `package.json` file.

I'm OK with this change, just saying :smiley:

Comment (mareklibra) [reply frontend/packages/kubevirt-plugin/package.json]: I know, but this is faster and clear at the first glance ...

I'm OK with your suggestion as well, just saying :-)

Comment (vojtechszocs) [reply frontend/packages/kubevirt-plugin/package.json]: Agreed it's more clear but only at code review time, when the change is introduced.

To see the exact resolution, one must still run `yarn why kubevirt-web-ui-components` or lookup the resolution entry in `yarn.lock` manually.

I think the best range would be `0.1.x` but that's just my personal preference.

**Pull request merged**